packages/kbn-reporting/common/license_check.ts

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I think this file is just used for `public` so it shouldn't be under a `common` location